### DISCUSSION TITLE
Can someone access the course slides?

### DISCUSSION THREAD

**Discussion by gagankhetwani** [cafawo/FinancialDataAnalytics#21] [Q&A] [answered]
I am getting an error while accessing the course slides. Is everyone facing the same issue?
![Screenshot 2024-05-19 124018](https://github.com/iwh-halle/FinancialDataAnalytics/assets/74928103/4e5e7f05-8b0c-48fe-94a5-d939bd1e7d2f)

**Comment by Big-Eric-Blip** [comment]
Same with me :(

**Comment by cafawo** [comment, marked as answer]
Which file did you try to open? As of this morning, I am able to view this file fine: https://github.com/iwh-halle/FinancialDataAnalytics/blob/master/slides_pt4.ipynb

![image](https://github.com/iwh-halle/FinancialDataAnalytics/assets/21214530/40a9559f-db0f-49c6-b7a0-a2a517acc66d)

![image](https://github.com/iwh-halle/FinancialDataAnalytics/assets/21214530/809bf301-8bd7-42f9-99dc-4c66a59f5926)

**Reply by gagankhetwani** [reply]
I tried opening all 4 files and none of them was working. But Now I can open all the files. Thanks.
